src/components/v5/common/ActionSidebar/partials/Motions/Motions.tsx

```diff
@@ -55,10 +55,20 @@ const Motions: FC<MotionsProps> = ({ transactionId }) => {
     networkMotionState === NetworkMotionState.Failed;
 
   useEffect(() => {
-    startPollingForAction();
-    setActiveStepKey(networkMotionState);
+    /** Not using motionFinished as it might also include the NetworkMotionState.Finalizable */
+    if (
+      [NetworkMotionState.Finalized, NetworkMotionState.Failed].includes(
+        networkMotionState,
+      )
+    ) {
+      stopPollingForAction();
+    } else {
+      startPollingForAction();
+    }
```

Comment:
So I don't think this works as you intended, as the list doesn't update properly when creating new agreements.

I'd say discard all of this, as it's really not part of this fix, and if you really want to fix this properly, use `subscriptions` instead of `queries`